src/components/modals/ExcludeClassTimeFilters.tsx

```diff
@@ -0,0 +1,247 @@
+                        <Stack direction={"column"} gap={2}>
+                            <InputLabel id="exclude-class-time-filter-weekday">Ukedag</InputLabel>
+                            <Select
+                                labelId="exclude-class-time-filter-weekday"
+                                id="exclude-class-time-select"
+                                value={excludedClassTimeWeekday}
+                                label="Ukedag"
+                                onChange={({ target: { value } }) => {
+                                    setExcludedClassTimeWeekday(() => value as number);
+                                }}
+                            >
+                                {weekdays.map((day, i) => (
+                                    <MenuItem key={i} value={i + 1}>
+                                        {day}
+                                    </MenuItem>
+                                ))}
+                            </Select>
+                            <Stack direction={"row"} gap={2} alignItems={"center"}>
+                                <TimePicker
+                                    label="Fra"
+                                    views={["hours", "minutes"]}
+                                    ampm={false}
+                                    value={excludedClassTimeStartTime}
+                                    onChange={(newValue) => setExcludedClassTimeStartTime(() => newValue)}
+                                />
+                                <RemoveRounded />
+                                <TimePicker
+                                    label="Til"
+                                    views={["hours", "minutes"]}
+                                    ampm={false}
+                                    value={excludedClassTimeEndTime}
+                                    onChange={(newValue) => setExcludedClassTimeEndTime(() => newValue)}
+                                />
+                            </Stack>
+                            <Stack direction={"row"} gap={1} justifyContent={"center"}>
+                                <Button
+                                    startIcon={<Add />}
+                                    variant={"outlined"}
+                                    type={"submit"}
+                                    disabled={!inputValid}
+                                    size="large"
+                                    fullWidth
+                                >
+                                    Skjul tidsrom
+                                </Button>
+                            </Stack>
+                        </Stack>
+                    </FormGroup>
+                </FormControl>
+            </form>
+            <Divider />
+            <Stack>
+                <Typography variant="h6" textAlign={"center"} sx={{ fontSize: 18 }}>
+                    Skjulte tidsrom
+                </Typography>
+                <Box sx={{ mt: 1 }}>
```

Comment:
Would prefer consistent column width. Maybe use some grid solution?

<img width="395" alt="image" src="https://github.com/user-attachments/assets/33d9643d-7e92-42bc-b8ff-1c3be4faaf0f">

Reply:
I think the easiest is to just throw in some common percentage width on day and interval, like this:

```jsx
// ...
<Typography width={"40%"}>{weekdays[filter.weekday - 1]}</Typography>
<Typography width={"50%"}>
    {filter.startHour.toFixed(0).padStart(2, "0")}:
    {filter.startMinute.toFixed(0).padStart(2, "0")}
    {" – "}
    {filter.endHour.toFixed(0).padStart(2, "0")}:
    {filter.endMinute.toFixed(0).padStart(2, "0")}
</Typography>
// ...
```

Should be perfectly adequate 😎

Reply:
:white_check_mark: